src/translations/en.json

```
@@ -5571,7 +5571,7 @@
         },
         "core-config": {
           "intro": "Hello {name}, welcome to Home Assistant. How would you like to name your home?",
-          "intro_core": "We will be setting up some core configuration for your Home Assistant instance now, you can always change this later in the configuration panel.",
+          "intro_core": "We will set up the basics together. You can always change this later in the settings.",
           "intro_location": "We would like to know where you live. This information will help with displaying information and setting up sun-based automations. This data is never shared outside of your network.",
```

How about something like this in order to to give an intro to HA lingo:

```suggestion
          "intro_location": "Let's setup the location for your home zone. You will then be able to display and automate using local information like sunrise and sunset times, weather, or the proximity of your tracked devices. This data is never shared outside of your network, and you can always change it later in settings.",
```

That way users form an idea of how it's used right off the bat and can make the decision to be accurate or not.

---

What about a shorter version?
"Let's setup the location for your home zone. You will then be able to display location-based information such as the weather and set up sun-based and home presence automations. This data is never shared outside of your network."